AGENTS.md

```diff
@@ -0,0 +1,224 @@
+# RDoc Project Guide for AI Agents
+
+## Project Overview
+
+**RDoc** is Ruby's default documentation generation tool that produces HTML and command-line documentation for Ruby projects. It parses Ruby source code, C extensions, and markup files to generate documentation.
+
+- **Repository:** https://github.com/ruby/rdoc
+- **Homepage:** https://ruby.github.io/rdoc
+- **Required Ruby:** >= 2.6.0 (targets 3.0+)
```

Comment:
Do you find that this helps AI use only 2.6 valid features? My only concern is that this is almost guaranteed to go stale when the minimum is bumped 😂

Reply:
I'll ask it to read gemspec to avoid becoming stale